core/src/main/java/com/redhat/lightblue/client/response/DefaultLightblueDataResponse.java

```diff
@@ -51,30 +51,6 @@ public boolean hasDataErrors() {
         return err != null && !(err instanceof NullNode) && err.size() > 0;
     }
 
-    @Deprecated
-    @Override
-    public boolean hasError() {
```

Comment:
This change is being made long after version bump, a little concerned it will break the build.